### ISSUE TITLE
Keyboard focus order is not logical in react flow screen using tab key in windows.

### ISSUE THREAD

**Issue by msftedad** [xyflow/xyflow#5659]
### What platform were you using when you found the bug?

- React Flow version: 12.10.0
- OS: Windows 11 25H2 (OS Build 26200.6584)
- Browser: Edge (Version 141.0.3537.57)

### Live code example

 https://play.reactflow.dev/

### Describe the Bug

**Test Environment:**
OS: Windows 11 25H2 (OS Build 26200.6584)
Browser: Edge (Version 141.0.3537.57)
URL: https://play.reactflow.dev/

**User Impact:**
Keyboard users can’t move through the screen in the right order using the Tab key, which makes it hard to use and confusing. This causes problems for people who rely on keyboard navigation, like screen reader users.

**Actual Result:**
Keyboard focus order is not logical in react flow screen using tab key in windows.

### Steps to reproduce the bug or issue

1. Open the URL: https://play.reactflow.dev/ in edge browser in windows.
2. 'React Flow Playground' page will appear.
3. Press tab key to reach playground section.
4. Observe whether keyboard focus order is logical in react flow screen using tab key in windows.

### Expected behavior

Keyboard focus order should be in proper order which is from left to right and top to bottom in react flow screen.

### Screenshots or Videos

https://github.com/user-attachments/assets/8aac08f2-ce22-405c-baf6-2ad81e122008
<img width="960" height="543" alt="Image" src="https://github.com/user-attachments/assets/8a5e03d8-a98f-44a4-96df-22ce8a8022ab" />

**Comment by moklick** [comment]
duplicate of #5189
